architecture/Threatmodel.md

```diff
@@ -0,0 +1,128 @@
+### Elevation of Privilege
+Niet van toepassing privileges/authenticatie worden in dit proces zoals nu bekend niet gebruikt.
+
+## 3. Het uitvoeren van een test en het doorgeven van een positieve test uitslag.
+### Spoofing
+De test en uitslagen worden verwerkt via een web applicatie. TLS encryptie dient te worden uitgevoerd. De OWASP richtlijnen voor veilige web applicaties, om risico's zoals sql injection, cross site scripting en dergelijke uit te sluiten.   
+### Tampering
+Toegang tot de databases waarin tests worden opgelagen dient goed te worden geregeld. Daarnaast dient het systeem van de TAN code's ook cryptografisch geanalyseerd te worden. 
+### Non Repudiation 
+Gebruikers kunnen een behoorlijk belang hebben bij een negatieve test uitslag omdat een postieve verplicht tot quarantaine. Indien er feedback 
+### Information Disclosure
+Informatie over de test uitslag is privacy gevoelig en dient te worden afgeschermd. 
+### Denial of Service
+Het doen van de test en dan het weer doorgeven van een positieve test gebruiken ook een webapplicatie. Een denial of serive attack zou hier mogelijk kunnen zijn. 
+### Elevation of Privilege 
+Het stuk waarin informatie wordt opgeslagen dient gebruikers te authoriseren en kent mogelijk meerde privileges. Ook een functionaliteit om mogelijk een fout-positieve of fout-negatieve test later weer te corrigeren is een stuk van de applicatie waarin risico's worden gelopen. Er kan druk worden uitgeoefend om test resultaten later onterecht van uitslag te veranderen.  
+## 4. Installatie en onboarding Risco's 
+###  Information Disclosure
+Informatie over de installatie van uit de App Store/ PlayStore is bekend bij Apple/Google en statistieken zijn hierover te verkrijgen.
+### Non Repudiation 
+Een druk om de app te installeren in bepaalde iets meer riskante bijeenkomsten (denk aan horeca, sport of OV)  is te verwachten.  
+  ### Denial of Service
+Niet van toepassing, dit licht vooral bij App Store/ Playstore beheerders. 
+### Elevation of Privilege
+De APP moet niet meer privileges gebruiken dan nodig. De privileges waar toestemming voor gevraagd wordt, moeten worden uitgelegd.   
+## 5. Overige app/ gebruiker interacties. 
+Dit is afhanklijk van de invulling van deze interacties. Te denken valt aan een test modus, extra informatie over Corona die getoond wordt aan de gebruiken.
+## Tegenstanders 
+### Script Kiddies. 
+Te denken valt aan jongeren die gebruiken maken van standaard programmatuur om zwakheden te ontdekken. Ze zijn meestal uit Nederland, vaak nog minderjarig. Een bekende klasse aanvallen die gebruikt worden zijn Denial of Service Attacks. Dit om de service te verstoren. Een voorbeeld hiervan zijn DDOS aanvallen op de internet betaal sites van de banken in 2018. 
```

Comment:
"Ze zijn meestal uit Nederland" = afkomstig uit NL, of juist buiten Nederland te vinden? Verder misschien niet helemaal relevant om die bevolkingsgroep zo specifiek te benoemen; het gaat immmers om hun gedrag en werkwijze voor zover dit model er op in zou moeten gaan :smile:

Reply:
De bewoording iets aangepast. De threat actors zijn een beetje stereotyp neergezet. Het is waarschijnlijk, maar geen zekerheid dat Nederlandse skript kiddies zich richten op Nederlandse apps.